src/java/us/kbase/templates/module_test_perl_client.vm.properties

```diff
@@ -1,28 +1,27 @@
 use strict;
 use warnings;
+use feature qw( say );
 
-use Data::Dumper;
-use Test::More;
+use Test::Most;
 use Config::Simple;
 use Time::HiRes qw(time);
 use Bio::KBase::AuthToken;
 use installed_clients::WorkspaceClient;
 #if ($example)
 use installed_clients::AssemblyUtilClient;
 #end
-use ${module_name}::${module_name}Impl;
+
+require_ok "${module_name}::${module_name}Impl";
 
 local $| = 1;
 my $token        = $ENV{ 'KB_AUTH_TOKEN' };
 my $config_file  = $ENV{ 'KB_DEPLOYMENT_CONFIG' };
 my $callback_url = $ENV{ 'SDK_CALLBACK_URL' };
 
 my $config       = Config::Simple->new( $config_file )->get_block( '${module_name}' );
-my $ws_url    = $config->{ "workspace-url" };
```

Comment:
Has ws_url become unnecessary, or was it always superfluous? I don't see any edits related top it beyond removing it's declaration

Reply:
I've just moved it into the `get_ws_client` sub (a couple of lines down), as it is only needed for initialising the workspace client.

Reply:
Ok, I see it now. I have another question though.